modules/swagger-parser-v3/src/test/java/io/swagger/v3/parser/test/OpenAPIV3ParserTest.java

```diff
@@ -1759,7 +1759,6 @@ public void testValidationIssueFalseNegative() {
         ParseOptions parseOptions = new ParseOptions();
         parseOptions.setResolveFully(true);
         SwaggerParseResult result = new OpenAPIV3Parser().readLocation("src/test/resources/validation/path-parameter-validation-ref-invalid.yaml", null, parseOptions);
-
         // should detect, but validation misses the error of the content in the ref, in order to avoid false positive
```

Comment:
Hi, what is this test doing? there are no assertion. please remove the comments.